nx_cugraph/utils/misc.py

```diff
@@ -58,6 +59,12 @@ def pairwise(it):
         "in the algorithm. If None, then dtype is determined by the edge values."
     ),
 }
+outbound_attraction_distribution = {
+    "bool": (
+        "Distributes attraction along outbound edges"
```

Comment:
I'm assuming a "." (or actually, a ". ") is needed for when the strings are concatenated in the docs generated from this?
```suggestion
        "Distributes attraction along outbound edges."
```